### PR TITLE
hard hats now protect you from headbutting airlocks while retarded

### PR DESCRIPTION
**Pull request by Spacedong**
the most important PR and the pinnacle of my coding skill

### CODE REVIEW

**Comment by Spacedong** [comment]
travis i swear

**Comment by ArcLumin** [comment]
Pinnacle of skill

but check failed

**Comment by JohnGinnane** [comment]
I think the error is:

`if(!istype(H.head, /obj/item/clothing/head/helmet || if(!istype(/obj/item/clothing/head/hardhat))`

**Comment by Spacedong** [comment]
so how do I fix it nigger

**Comment by JohnGinnane** [comment]
`if(!(istype(H.head, /obj/item/clothing/head/helmet || istype(/obj/item/clothing/head/hardhat)))`

**Comment by JohnGinnane** [comment]
> the pinnacle of my coding skill

okay

**Comment by Spacedong** [comment]
im coming for travis

File: code/game/machinery/doors/airlock.dm [hunk 1]
```diff
@@ -552,7 +552,7 @@ About the new airlock wires panel:
 		var/mob/living/carbon/human/H = user
 		if(H.getBrainLoss() >= 60)
 			playsound(src.loc, 'sound/effects/bang.ogg', 25, 1)
-			if(!istype(H.head, /obj/item/clothing/head/helmet))
+			if(!(istype(H.head, /obj/item/clothing/head/helmet || istype(/obj/item/clothing/head/hardhat)))
```
Comment (JohnGinnane) [inline code/game/machinery/doors/airlock.dm]: `istype(/obj/item/clothing/head/hardhat)` should be `istype(H.head, /obj/item/clothing/head/hardhat)`

Comment (JohnGinnane) [reply code/game/machinery/doors/airlock.dm]: Also missing closing parenthesis for first istype()

**Comment by oprayx73** [comment]
n'dooru

**Comment by JohnGinnane** [comment]
Code looks good now, although I kinda wrote it for him..

**Comment by ArcLumin** [comment]
To make sure you don't bust your head open on a door because you're retarded, you must first not bust your head on your code because you're retarded

**Comment by Spacedong** [comment]
LMAO

**Comment by TheOneAndOnlyCreeperJoe** [comment]
I see Spacedong's Cluwne ban hasn't expired yet.

**Comment by Hackole** [comment]
sure why not +1

**Comment by McBawbaggings** [comment]
Yeah okay +1

**Pull request merged**